### DISCUSSION TITLE
Unexpected token ) for function call

### DISCUSSION THREAD

**Discussion by SimonChaumet** [eslint/eslint#13708] [General] [answered]
**Tell us about your environment**
Environment Info:

Node version: v10.17.0
npm version: v6.11.3
Local ESLint version: v7.9.0 (Currently used)
Global ESLint version: Not found

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**
ecmaVersion: 6
sourceType: "script"

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  env: {
    commonjs: true,
    es6: true,
    node: true
  },
  extends: [
    'eslint:recommended'
  ],
  parserOptions: {
    ecmaVersion: 6,
    sourceType: "script",
  },
  rules: {
    'no-console': 'warn',
    'no-debugger': 'warn',
    'block-scoped-var': "warn",
    'no-multi-spaces': "warn",
    'array-bracket-newline': ["warn", "consistent"],
    'block-spacing': "warn",
    'brace-style': ["warn", "1tbs"],
    'comma-dangle': ["warn", "always-multiline"],
    'comma-spacing': ["warn", {before: false, after: true}],
    'comma-style': ["warn", "last"],
    'eol-last': ["warn", "always"],
    'func-call-spacing': ["warn", "never"],
    'indent': ["error", 4],
    'key-spacing': ["warn", {mode: "minimum", afterColon: true}],
    'keyword-spacing': ["warn", {before: true, after: true}],
    'linebreak-style': ["warn", "unix"],
    'new-parens': ["warn", "always"],
    'newline-per-chained-call': ["warn", {ignoreChainWithDepth: 3}],
    'no-lonely-if': "warn",
    'no-trailing-spaces': "warn",
    'no-unneeded-ternary': "warn",
    'no-whitespace-before-property': "warn",
    'semi-spacing': ["warn", {before: false, after: true}],
    'space-before-blocks': "warn",
    'space-before-function-paren': ["warn", {anonymous: "never", named: "never", asyncArrow: "always"}],
    'space-infix-ops': ["warn", {"int32Hint": true}],
    'space-unary-ops': ["warn", {words: true, nonwords: false}],
    'spaced-comment': ["warn", "always"],
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
exports.updateActions = function(req, res) {
    /* Must have only the changed actions in parameters to be performant TODO IN FRONT*/
    const promises = []
    for (const action of req.body) {
        promises.push(
            models.Action.update(action, {
                where: {
                    actionId: action.actionId,
                },
            }), // Problem is here
        )
    }
    Promise.all(promises).then(() => {
        return res.sendStatus(200);
    }, err => {
        logger.error('%o', err);
        res.sendStatus(500);
    })
}
```

```bash
eslint app/ --ext .js
```

**What did you expect to happen?**
No error found

**What actually happened? Please include the actual, raw output from ESLint.**
> eslint app/ --ext .js

/home/simonc/projects/sx-projects-back/app/controllers/actionController.js
  33:9  error  Parsing error: Unexpected token )

✖ 1 problem (1 error, 0 warnings)

If I remove the comma (see line "Problem is here") the error disappear but I would like to keep this comma because node understand it correctly and it give better commits (like comma-dangle rule)

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by anikethsaha** [comment, marked as answer]
Thanks for the issue. 


Use `ecmaversion : 2017` or higher.

**Comment by yeonjuan** [comment]
@SimonChaumet 
The trailing comma in function parameter is allowed in ECMAScript 2017. - [trailing comma-MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas). 
> Use ecmaversion : 2017 or higher.

so [it](https://github.com/eslint/eslint/issues/13708#issuecomment-696694989) can help you
